docs/src/pages/customization/palette/palette.md

```diff
@@ -4,7 +4,7 @@
 
 ## Palette colors
 
-A color intention is a mapping of a palette color to a given intention within your application.
+A "palette color" is a mapping of colors to a given intention within your application.
```

Comment:
I thought we decided to drop this sentence?

Reply:
So we won't have this one at all? I'm confused :(